samples/XML/Asset.workflow

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could we replace this with a bit of a longer sample file?

Reply:
Sure. I didn't as the rest of the content is never considered and this sample is incredibly popular already 😄

Reply:
Done.

Reply:
Sure, it's not critical. The content of `.workflow` XML files might be considered if the file doesn't have a root tag though, so it's important that we train the classifier in case that happens. It's not a very frequent case so we should be fine. Tiny files are also a pain when/if we later add a heuristic rule and have to manually exclude all tiny files from the tests.